src/index.js

```diff
@@ -100,42 +102,45 @@ sequelize.sync({ force: isTest || isProduction }).then(async () => {
 });
 
 const createUsersWithMessages = async date => {
-  await models.User.create(
+  try {
+  const user1 = await models.User.create(
```

Comment:
Suggestion: remove declaration of user1 and user2 if the variables are not needed afterward.

Reply:
removed unused variable